src/components/tag/Tag.stories.mdx

```diff
@@ -0,0 +1,28 @@
+import { ArgsTable, Canvas, Meta, Story } from '@storybook/addon-docs/blocks';
```

Comment:
fine for now (since i have not done that for my PRs lately as well) ... just keep in mind to ad the docs later on

Reply:
I usually do but I couldn't find docs for the tags, not detailed like the ones for text input for example.. waiting to align with design on that